docs/Options-in-Constructor.md

```diff
@@ -207,7 +208,7 @@ Learn more: [Change Fonts](https://surveyjs.io/Documentation/Pdf-Export?id=Custo
 
 Obsolete.  
 
-- [useCustomFontInHtml](https://github.com/surveyjs/survey-pdf/blob/1220a71b51daddf1c4c8d506382c50be5f1b2941/src/doc_controller.ts#40)  
+- [useCustomFontInHtml](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#40)  
 `useCustomFontInHtml?: boolean;`  
 Notes:  
 Earlier was used together with [base64Normal](base64Normal) and [base64Bold](base64Bold) to specify whether [custom fonts](https://surveyjs.io/Documentation/Pdf-Export?id=Customization-ChangeFonts#use-custom-font) (loaded throught the `base64Normal` and `base64Bold` options) should apply to texts rendered within questions of the HTML type.  
```

Comment:
This sentence needs a rewrite.
You have a word missing right at the beginning: what was used together...?
Also, odd phrasing on "texts rendered within questions of the HTML type"